tests/foreman/ui/test_rhc.py

```diff
@@ -175,6 +175,6 @@ def test_positive_configure_cloud_connector(
 
     assert rhc_status.status == 0
     assert "Connected to Red Hat Subscription Management" in rhc_status.stdout
-    assert "The Red Hat connector daemon is active" in rhc_status.stdout
+    assert "The Remote Host Configuration daemon is active" in rhc_status.stdout
```

Comment:
@jameerpathan111 In the PRT results it seems, this change wasn't required for 6.12.z/6.11.z, can you check and update PR?

Reply:
It's because Satellite 6.13 was installed on rhel 8.8 and 6.12/6.11 was installed on rhel 8.7
It'll pass once we have a new z stream snap for those releases.

Reply:
Ah okay, I believe you, so let's merge it as is